.rubocop.yml

```diff
@@ -1,5 +1,7 @@
 AllCops:
   TargetRubyVersion: 3.1.1
+Metrics/AbcSize:
+  Max: 20
```

Comment:
Likely to be temporary, but a small change triggered this cop and it wasn't worth investigating for a minor change. Therefore, I should look into optimizing this in the future.